.github/workflows/copy-import-script.yml

```diff
@@ -31,19 +23,23 @@ jobs:
       - name: Compile the script
         run: deno compile --allow-env --allow-net --allow-run --target x86_64-unknown-linux-gnu --output target/importBoardsTxt scripts/importBoardsTxt.ts -- --githubToken ${{ secrets.IMPORT_SCRIPT_ACCESS_TOKEN }}
 
-      - name: Copy the script to hosts
+      - name: Extract branch name
+        run: echo "name=$(echo ${GITHUB_REF#refs/heads/})" >>$GITHUB_OUTPUT
+        id: branch_name
+
+      - name: Copy the script to host
         uses: appleboy/scp-action@master
         with:
-          host: ${{ matrix.host }}
+          host: ${{ steps.branch_name.outputs.name }}.staging.kitspace.dev
```

Comment:
There seems to be a `${{ github.ref_name }}` now which we should probably start using as it's a bit simpler.